pkg/authorization/authorizer/authorizer.go

```diff
@@ -27,17 +27,17 @@ func NewSubjectLocator(delegate authorizerrbac.SubjectLocator) SubjectLocator {
 	return &openshiftSubjectLocator{delegate: delegate}
 }
 
-func (a *openshiftAuthorizer) Authorize(attributes authorizer.Attributes) (bool, string, error) {
+func (a *openshiftAuthorizer) Authorize(attributes authorizer.Attributes) (authorizer.Decision, string, error) {
 	if attributes.GetUser() == nil {
-		return false, "", errors.New("no user available on context")
+		return authorizer.DecisionDeny, "", errors.New("no user available on context")
 	}
-	allowed, delegateReason, err := a.delegate.Authorize(attributes)
-	if allowed {
-		return true, reason(attributes), nil
+	authorized, delegateReason, err := a.delegate.Authorize(attributes)
```

Comment:
this should return authorized, not convert it to Deny

Reply:
> this should return authorized, not convert it to Deny

done